lib/internal/modules/esm/loader.js

```diff
@@ -148,26 +148,28 @@ function nextHookFactory(chain, meta, { validateArgs, validateOutput }) {
   }
 
   return ObjectDefineProperty(
-    async (...args) => {
+    async (arg0, context) => {
```

Comment:
Note that this change the `.length` of the helper function, maybe we want instead:

```suggestion
    async (arg0 = undefined, context) => {

Reply:
huh? I dunno what you mean, but it doesn't hurt and you're usually, right so I'll just assume there's a good reason (but I'm curious to know).

Reply:
```js
console.log(((...args) => {}).length); // 0
console.log(((arg0, context) => {}).length); // 2
console.log(((arg0 = undefined, context) => {}).length); // 0
```

Reply:
🤯 TIL!